R/tm_outliers.R

```diff
@@ -109,29 +116,7 @@ ui_outliers <- function(id, ...) {
   is_single_dataset_value <- teal.transform::is_single_dataset(args$outlier_var, args$categorical_var)
 
   teal.widgets::standard_layout(
-    output = teal.widgets::white_small_well(
-      uiOutput(ns("total_outliers")),
-      DT::dataTableOutput(ns("summary_table")),
-      uiOutput(ns("total_missing")),
-      br(), hr(),
-      tabsetPanel(
-        id = ns("tabs"),
-        tabPanel("Boxplot", teal.widgets::plot_with_settings_ui(id = ns("box_plot"))),
-        tabPanel("Density Plot", teal.widgets::plot_with_settings_ui(id = ns("density_plot"))),
-        tabPanel("Cumulative Distribution Plot", teal.widgets::plot_with_settings_ui(id = ns("cum_density_plot")))
-      ),
-      br(), hr(),
-      teal.widgets::optionalSelectInput(
-        inputId = ns("table_ui_columns"),
-        label = "Choose additional columns",
-        choices = NULL,
-        selected = NULL,
-        multiple = TRUE
-      ),
-      h4("Outlier Table"),
-      teal.widgets::get_dt_rows(ns("table_ui"), ns("table_ui_rows")),
-      DT::dataTableOutput(ns("table_ui"))
-    ),
+    output = uiOutput(ns("output_whole")),
```

Comment:
Changing one of the encodings regenerate the whole output (tabs). For example when user explores "density plot" after encoding change the whole module gets regenerated.

Reply:
Yes, but otherwise we get this:
![image](https://user-images.githubusercontent.com/114988527/210758182-df47ddfc-8e57-4536-87e6-f8f13fb9bdc7.png)

Reply:
I think that's ok to be honest or if you prefer put a req at the top and move the validation into each of the tabs?
![image](https://user-images.githubusercontent.com/15201933/210765941-63043d57-8602-4136-bce9-89cd20d06fbe.png)

Reply:
The think is they are all always displayed (well, the top one not necessarily). You will always get the Outlier Table and the tabs. Displaying the messages more than once (in output) doesn't make much sense, does it? 
And in the image above, everything is already `req`-ed.

To me there is no ideal solution here. The question is: which one do we prefer?

Reply:
If I was a user I'd prefer to not re/render and switch the tab. This output is too reactive, everything blinks on any encoding change which.

> Displaying the messages more than once (in output)

What do you mean by this. Where the messages are displayed again?

Reply:
> move the validation into each of the tabs?

I musunderstood "tabs" here as horizontal sections of the output.

Reply:
Is this better?
![image](https://user-images.githubusercontent.com/114988527/210801456-8cc80418-eec4-4047-b80e-c1e474d22516.png)

Reply:
yes